cli_reference/openshift_developer_cli/odo-release-notes.adoc

```diff
@@ -5,10 +5,95 @@ include::modules/common-attributes.adoc[]
 :context: odo-release-notes
 toc::[]
 
-include::modules/openshift-developer-cli-odo-notable-features.adoc[leveloffset=+1]
+[id="odo-notable-improvements_{context}"]
+== Notable improvements in {odo-title}{nbsp}{odo-ver}
 
-include::modules/openshift-developer-cli-odo-getting-support.adoc[leveloffset=+1]
+* IBM Z and PowerPC architecture binaries are now available.
+* `odo catalog` has been improved to provide more useful output information.
+* Information prompts have been added to the `odo service create` command, providing possible next step scenarios.
+* Log verbosity can now be set with an environment variable `ODO_LOG_LEVEL`.
+* Use `odo preference set PushTimeout <seconds>` to specify the number of seconds `odo` waits for a component pod to be deployed before the command fails.
+* Overall documentation improvements. The documentation now includes a description of {odo-title}'s inner architecture.
 
-include::modules/openshift-developer-cli-odo-technology-preview.adoc[leveloffset=+1]
+////
 
-include::modules/openshift-developer-cli-odo-known-issues.adoc[leveloffset=+1]
\ No newline at end of file
+= Notable features in odo for the OCP 4.2 release
+
+{odo-title}{nbsp}{odo-ver} offers a simple way to develop applications on {product-title}. {odo-title} is completely client-based and requires no server within {product-title} cluster.
+
+{odo-title}{nbsp}{odo-ver} comes with:
+
+* Simple syntax and design centered around concepts familiar to developers such as projects, applications and components.
+* Compatibility with any language or runtime within the OpenShift catalog of component types.
+* Capability to add custom component types using custom image builders.
+* Official support for Java and Node.js component types.
+* Service catalog which allows users to connect applications deployed on {product-title} cluster to the Template Service Broker.
+* `odo service create` - an interactive mode which guides users through the process of creating applications.
+* `odo watch` - a feature that automatically detects changes to local code and applies the changes to the component in real time.
+
+////
+
+[id="odo-getting-support_{context}"]
+== Getting support
+
+.For Documentation
+
+If you have found an error or have suggestions for improving the documentation, file an issue in link:http://bugzilla.redhat.com[Bugzilla]. Choose the *{product-title}* product type and the *Documentation* component type.
+
+.For Product
+
+If you have found an error, encountered a bug or have suggestions for improving the functionality of {odo-title}, file an issue in link:http://bugzilla.redhat.com[Bugzilla]. Choose the *Red Hat odo for OpenShift Container Platform* product type.
```

Comment:
s/have found/find
s/a bug or/a bug, or
Also, you may want to use {product-title} instead of spelling out OCP, even though it's a UI field.